src/web/components/ApiKeyManagement/KeyHelper.ts

```diff
@@ -1,12 +1,12 @@
 import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
 
-const yearInSeconds = 31556952;
+export const yearInSeconds = 60 * 60 * 24 * 365.25;
 
-export const needsRotating = (apiKey: ApiKeyDTO): boolean => {
-  const currentDate = new Date().getTime();
+// api key is recommended to rotate if older than one year
+export const shouldRotateApiKey = (apiKey: ApiKeyDTO): boolean => {
```

Comment:
sorry - I'm now thinking that this method could include the `disabled` check, rather than doing the two checks inside `getApiKeysToRate` in `home.tsx` - that way all the logic lives in the same place and we can more easily cover with tests & stories. Does that seem reasonable?

Reply:
Yep sounds good - I've added two more test cases to check the disabled key as well